commit 7565e9fd7fe1dba00d9acba3ec1a1714da3ee942
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:22:24 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-proper-sqlite3_libs-definition.diff

diff --git a/Mk/macports.autoconf.mk.in b/Mk/macports.autoconf.mk.in
index 2b76c8c75..049057de9 100644
--- a/Mk/macports.autoconf.mk.in
+++ b/Mk/macports.autoconf.mk.in
@@ -27,7 +27,7 @@ TCL_STUB_LIB_SPEC	= @TCL_STUB_LIB_SPEC@
 LIBS			= @LIBS@
 READLINE_LIBS		= @READLINE_LIBS@
 MD5_LIBS		= @MD5_LIBS@
-SQLITE3_LIBS		= @abs_top_builddir@/@VENDOR_DESTROOT@@TCL_PREFIX@/lib/@SQLITE3_LIBNAME@/lib@SQLITE3_LIBNAME@$(SHLIB_SUFFIX)
+SQLITE3_LIBS		= -L@abs_top_builddir@/@VENDOR_DESTROOT@@TCL_PREFIX@/lib/@SQLITE3_LIBNAME@/ -l@SQLITE3_LIBNAME@ -Wl,-rpath,@TCL_PACKAGE_PATH@/@SQLITE3_LIBNAME@
 CURL_LIBS		= @LDFLAGS_LIBCURL@
 INSTALL			= @INSTALL@
 MTREE			= @MTREE@
